README.md

```diff
@@ -38,10 +49,24 @@ This could be to write an article, code up a tool, create a training video, etc.
 1. **Validated**.  More than one company is using the solution.
 
 We have a regular meeting where we will list new challenges, discuss proposed solutions, and demo finished solutions.
-It is expected that the implementation of many solutions may be contributions to other working groups in the InnerSource Commons (e.g. Patterns, Learning Path).
+It is expected that the implementation of many solutions may be contributions to other working groups in the InnerSource Commons (e.g. [Patterns][], [Learning Path][]).
 
 ## Recording
 
 Check out this presentation explaining the motivation, tasks, and goals of this working group.
 
 [<img src="https://user-images.githubusercontent.com/9609562/211604583-ee41a7b4-cb56-4f72-9256-bfa2a560258b.png" width="50%" />](https://www.youtube.com/watch?v=r8Ce7GlwBeA)
+
+## Refernnces
+
+* [Working Group Charter](https://github.com/InnerSourceCommons/foundation-governance/edit/master/ispo-working-group-charter.md)
```

Comment:
Is this needed anymore?  Most of the material is duplicated here.

Reply:
Probably not. I just added it because I was looking for it and didn't have the link handy.

We could leave the reference here as a reminder that we have to clean up this duplication eventually?

Reply:
Sure